### PR TITLE
Add demographic icons

### PR DESCRIPTION
**Pull request by DanielHeath**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
     "serialize-javascript": "1.1.2",
     "sessionstorage": "0.0.1",
     "style-loader": "0.12.4",
+    "svg-inline-loader": "^0.3.0",
```
Comment (danni) [inline package.json]: Remove `^`

**Comment by danni** [comment]
Out of interest, can we just have webpack do the loading and inlining for us?

**Comment by DanielHeath** [comment]
Yes, but you end up having to use webpack to build the server code.

**Comment by danni** [comment]
++ if tests pass

**Pull request merged**